src/main/webapp/paginationScript.js

```diff
@@ -85,9 +88,47 @@ function locateImageElement(card) {
 }
 
 function loadImage(listingImage, photoReference) {
-  const KEY = 'REDACTED';
   let maxwidth = 400;
 
   listingImage.src = 'https://maps.googleapis.com/maps/api/place/photo?photoreference=' 
         + photoReference + '&key=' + KEY + '&maxwidth=' + maxwidth;
+}
+
+function locateWebsiteUrlElement(card) {
+  return card.getElementsByTagName('button')[0];
```

Comment:
What about grabbing the element by className? This is highly susceptible to any button placement changes in the DOM.

Reply:
changed!